packages/react-instantsearch-hooks-dom/src/ui/Breadcrumb.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+import { isModifierClick } from './lib/isModifierClick';
+
+import type { useBreadcrumb } from 'react-instantsearch-hooks';
+
+export type BreadcrumbTranslations = {
+  /**
+   * The label of the root element.
+   */
+  root: string;
+};
+
+export type BreadcrumbClassNames = {
+  /**
+   * Class names to apply to the root element
+   */
+  root: string;
+  /**
+   * Class names to apply to the root element, when there's no refinement possible
+   */
+  rootNoRefinement: string;
+  /**
+   * Class names to apply to the list element
+   */
+  list: string;
+  /**
+   * Class names to apply to each item element
+   */
+  item: string;
+  /**
+   * Class names to apply to a selected breadcrumb element
+   */
+  itemSelected: string;
+  /**
+   * Class names to apply to the separator between breadcrumb elements
+   */
+  separator: string;
+  /**
+   * Class names to apply to the link element
+   */
+  link: string;
+};
+
+type UseBreadcrumbRenderState = ReturnType<typeof useBreadcrumb>;
+
+export type BreadcrumbProps = React.HTMLAttributes<HTMLDivElement> &
+  Pick<UseBreadcrumbRenderState, 'items' | 'createURL'> & {
+    classNames?: Partial<BreadcrumbClassNames>;
+    hasItems: boolean;
+    onNavigate: UseBreadcrumbRenderState['refine'];
+    separator?: string;
+    translations: BreadcrumbTranslations;
+  };
+
+export function Breadcrumb({
+  classNames = {},
+  items = [],
+  hasItems,
+  createURL,
+  onNavigate,
+  separator = '>',
+  translations,
+  ...props
+}: BreadcrumbProps) {
+  const handleClick =
+    (value: BreadcrumbProps['items'][0]['value']) =>
```

Comment:
```suggestion
    (value: BreadcrumbProps['items'][number]['value']) =>
```

```suggestion
    (value: BreadcrumbItem['value']) =>
```

```suggestion
    (value: string) =>
```


I don't really mind which one of these is picked

Reply:
Slight preference for option 2 (`BreadcrumbItem['value']`).

Reply:
`BreadcrumbConnectorParamsItem` is not well exported, not sure that's a good idea to import it as is:
```
import type { BreadcrumbConnectorParamsItem } from 'instantsearch.js/es/connectors/breadcrumb/connectBreadcrumb';
```

I can create a local type that maps to `UseBreadcrumbRenderState['items'][number]`, which can bring more legibility to the code, if that's ok.